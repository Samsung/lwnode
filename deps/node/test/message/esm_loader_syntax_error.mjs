// Flags: --experimental-loader ./test/fixtures/es-module-loaders/syntax-error.mjs
import '../common/index.mjs';
console.log('This should not be printed');
