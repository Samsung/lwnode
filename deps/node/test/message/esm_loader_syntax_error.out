(node:*) ExperimentalWarning: --experimental-loader is an experimental feature. This feature could change at any time
(Use `* --trace-warnings ...` to show where the warning was created)
file://*/test/fixtures/es-module-loaders/syntax-error.mjs:2
await async () => 0;
^^^^^^^^^^^^^

SyntaxError: Malformed arrow function parameter list
    at Loader.moduleStrategy (internal/modules/esm/translators.js:*:*)
    at async link (internal/modules/esm/module_job.js:*:*)
